dojson/utils.py

```diff
@@ -45,11 +45,14 @@ def wrapper(self, key, value, **kwargs):
 
 
 def filter_values(f):
-    """Remove None values from dictionary."""
+    """Remove None or empty values from dictionary."""
     @functools.wraps(f)
     def wrapper(*args, **kwargs):
         out = f(*args, **kwargs)
-        return dict((k, v) for k, v in six.iteritems(out) if v is not None)
+        out = dict((k, v) for k, v in six.iteritems(out) if v or v is False or v == 0)
```

Comment:
Actually this is the very point being debated in the whole PR
IMHO we need to filter as well empty dict, empty list and empty string, since they don't convey a meaning. Only `False`, and `0` are instead providing information.